### PR TITLE
feat: Create RouteVisualization component for external applications usage.

### PR DESCRIPTION
**Pull request by mmelko**

### CODE REVIEW

File: packages/ui/src/external/RouteVisualisation/RouteVisualization.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <RuntimeProvider catalogUrl={catalogUrl}>
+      <VisibleFlowsProvider>
+        <Visualization className={'canvas-page'} entities={visualEntities} />
+      </VisibleFlowsProvider>
+    </RuntimeProvider>
+  );
+};
+
+export const RouteVisualization: React.FC<{ catalogUrl: string; code: string; codeChange: (code: string) => void }> = ({
```
Comment (lordrip) [inline packages/ui/src/external/RouteVisualisation/RouteVisualization.tsx:18]: [curious-question] why the split component? :smile:

File: packages/ui/tsconfig.esm.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
       "@testing-library/jest-dom",
     ]
   },
-  "include": ["src/public-api.ts", "src/models-api.ts", "src/testing-api.ts", "src/global.d.ts"],
+  "include": ["src/public-api.ts", "src/models-api.ts", "src/testing-api.ts","src/components-api.ts", "src/global.d.ts"],
```
Comment (lordrip) [inline packages/ui/tsconfig.esm.json:31]: Should we add this on the other tsconfig.json as well?

Comment (mmelko) [reply packages/ui/tsconfig.esm.json:31]: isn't that implicitly inherited by ` "extends": "./tsconfig.esm.json",` ?

File: packages/ui/src/external/RouteVisualisation/RouteVisualization.tsx (hunk 1 above)
Comment (mmelko) [reply packages/ui/src/external/RouteVisualisation/RouteVisualization.tsx:18]: For reusing EntityProvider and RuntimeProvider. If there is an another way, I'm happy to refactor :)

File: packages/ui/tsconfig.esm.json (hunk 2 above)
Comment (lordrip) [reply packages/ui/tsconfig.esm.json:31]: Yes, you're right. [Here's the extension](https://github.com/KaotoIO/kaoto/blob/c5545336f8304d91e11c62c9c3ed3c7ae09b9f25/packages/ui/tsconfig.cjs.json#L2)

**Review by lordrip — APPROVED** [review]

**Pull request merged**